xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -175,7 +175,7 @@ void CPeripheralCecAdapter::Announce(AnnouncementFlag flag, const char *sender,
   else if (flag == GUI && !strcmp(sender, "xbmc") && !strcmp(message, "OnScreensaverActivated") && m_bIsReady)
   {
     // Don't put devices to standby if application is currently playing
-    if ((!g_application.m_pPlayer->IsPlaying() && !g_application.m_pPlayer->IsPaused()) && m_configuration.bPowerOffScreensaver == 1)
+    if (!g_application.m_pPlayer->IsPausedPlayback() && m_configuration.bPowerOffScreensaver == 1)
```

Comment:
Sorry, haven't checked recent internal API changes. it should not send the computer to standby if video is "active" (playing or paused) because users don't expect that to happen. It should only do this when the system is actually idle.